### PR TITLE
Add Azure configuration file

### PR DESCRIPTION
**Pull request by namurphy**
I am adapting this from PlasmaPy's Azure configuration file.

### CODE REVIEW

**Comment by StanczakDominik** [comment]
Need a hand with this?

**Comment by namurphy** [comment]
Sure!  I'm not as familiar with how Azure tests are set up, so help would be appreciated to make sure I'm setting things up correctly.

**Comment by namurphy** [comment]
...though I'm also thinking that maybe it would be easier to retemplate this with [SunPy's package template](https://github.com/OpenAstronomy/packaging-guide) which takes care of a bunch of the configurations for Azure, Read the Docs, etc.

**Comment by StanczakDominik** [comment]
I think it's fine to do this on our own for now, then compare/contrast/contribute upstream. 

I think the trick here is adding a new project to the PlasmaPy org over at https://dev.azure.com/plasmapy/, then adding a new Pipeline at https://dev.azure.com/plasmapy/PlasmaPy-NEI/_build, which I'm doing right now. Seems important to pick an existing azure-pipelines.yml file, which is why I'm gonna merge this now. :smile:

**Comment by StanczakDominik** [comment]
I think you may need to sign up for the org at https://dev.azure.com/plasmapy/, too.

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  CI_BUILD_ID: $(Build.BuildId)
+  CI_BUILD_URL: "https://dev.azure.com/plasmapy/plasmapy/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp37-* cp38-*
+  CIBW_SKIP: "*-win32 *-manylinux1_i686"
+
+resources:
+  repositories:
+  - repository: OpenAstronomy
+    type: github
+    endpoint: PlasmaPy-NEI
```
Comment (StanczakDominik) [inline azure-pipelines.yml:13]: Turns out this should have been just `PlasmaPy`. It probably references the org, not the repo. Or something. ^^`

**Pull request merged**